src/main/kotlin/solve/main/MainView.kt

```diff
@@ -47,9 +70,105 @@ class MainView : View() {
     private val rightSidePanelViews =
         createSidePanelsViews(rightSidePanelTabs, SidePanelLocation.Right)
 
+    private val importFAB = mfxButton {
+        VBox.setMargin(this, Insets(5.0, 8.0,10.0,8.0))
+
+        val circle = Circle(this.layoutX + 28.0, this.layoutY + 28.0,28.0)
+        clip = circle
+        graphic = ImageView(importIcon)
+        prefHeight = 56.0
+        prefWidth = 56.0
+
+        style = "-fx-background-color: #${Style.secondaryColor}; -fx-background-radius: 28;"
+
+        action{
+            importAction()
+        }
+    }
+
+    private val pluginsButton = mfxButton("Plugins") {
```

Comment:
Don't show unused tabs (plugins, settings, help) before the better times (don't remove the code and icons)